From c592c312d4dd144fc36dd8197952f8fe5b2179d4 Mon Sep 17 00:00:00 2001
From: buildslave <ctbbot@intel.com>
Date: Wed, 2 Nov 2022 12:42:45 +0000
Subject: [PATCH] mixin updates

Remove libmd.so from additional native libraries
Remove libmd.so from additional native libraries for CTS tests as
libmd is disabled.
Ported from master PR(OAM-104325)
https://github.com/projectceladon/device-androidia-mixins/pull/1511

Tracked-On: OAM-104242
Mixin-Reviewed-On: https://github.com/projectceladon/device-androidia-mixins/pull/1515
Signed-off-by: Lu Yang A <yang.a.lu@intel.com>
Signed-off-by: Vasoya,Nikhilx <nikhilx.vasoya@intel.com>
---
 caas/extra_files/public-libraries/public.libraries.txt           | 1 -
 caas_cfc/extra_files/public-libraries/public.libraries.txt       | 1 -
 celadon_ivi/extra_files/public-libraries/public.libraries.txt    | 1 -
 celadon_tablet/extra_files/public-libraries/public.libraries.txt | 1 -
 4 files changed, 4 deletions(-)

diff --git a/caas/extra_files/public-libraries/public.libraries.txt b/caas/extra_files/public-libraries/public.libraries.txt
index d7fc8f2..28209b9 100644
--- a/caas/extra_files/public-libraries/public.libraries.txt
+++ b/caas/extra_files/public-libraries/public.libraries.txt
@@ -1,3 +1,2 @@
 # Specify additional native libraries
 # So that Apps can use Intel proprietary libraries
-libmd.so
diff --git a/caas_cfc/extra_files/public-libraries/public.libraries.txt b/caas_cfc/extra_files/public-libraries/public.libraries.txt
index d7fc8f2..28209b9 100644
--- a/caas_cfc/extra_files/public-libraries/public.libraries.txt
+++ b/caas_cfc/extra_files/public-libraries/public.libraries.txt
@@ -1,3 +1,2 @@
 # Specify additional native libraries
 # So that Apps can use Intel proprietary libraries
-libmd.so
diff --git a/celadon_ivi/extra_files/public-libraries/public.libraries.txt b/celadon_ivi/extra_files/public-libraries/public.libraries.txt
index d7fc8f2..28209b9 100644
--- a/celadon_ivi/extra_files/public-libraries/public.libraries.txt
+++ b/celadon_ivi/extra_files/public-libraries/public.libraries.txt
@@ -1,3 +1,2 @@
 # Specify additional native libraries
 # So that Apps can use Intel proprietary libraries
-libmd.so
diff --git a/celadon_tablet/extra_files/public-libraries/public.libraries.txt b/celadon_tablet/extra_files/public-libraries/public.libraries.txt
index d7fc8f2..28209b9 100644
--- a/celadon_tablet/extra_files/public-libraries/public.libraries.txt
+++ b/celadon_tablet/extra_files/public-libraries/public.libraries.txt
@@ -1,3 +1,2 @@
 # Specify additional native libraries
 # So that Apps can use Intel proprietary libraries
-libmd.so
-- 
2.38.1

